core/templates/domain/topic/topic-update.service.spec.ts

```diff
@@ -509,6 +618,40 @@ describe('Topic update service', function() {
     }]);
   });
 
+  it('should properly decrement subtopic ids of moved subtopics ' +
+  'when a newly created subtopic is deleted', function() {
```

Comment:
Follow existing indentation practices (2 spaces), here and elsewhere.

Reply:
@shavavo Before requesting a second review, please make sure that you have replied to all comments, per the  instructions in [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)? In particular, for comments that you've addressed, please reply "Done." -- this helps keep track of which comments have been addressed and which haven't.

It seems like this comment was not addressed, and in fact the indentation was done incorrectly for the rest of the multi-line it() blocks? Please fix it to follow the previous format (2-space indentation) unless there's a good reason not to do so.

Reply:
Done.